### ISSUE TITLE
t/sqlite.t started to fail (with newest SQL::Abstract?)

### ISSUE THREAD

**Issue by eserte** [alranel/DBIx-Lite#19]
My smoker systems started to report the following failure:
```
DBD::SQLite::db prepare failed: near "FROM": syntax error at t/sqlite.t line 24.
# Looks like your test exited with 2 before it could output anything.
t/sqlite.t ............... 
Dubious, test returned 2 (wstat 512, 0x200)
Failed 20/20 subtests 
```
I guess this is caused by changes in SQL::Abstract 2.x (@shadowcat-mst: FYI).

**Comment by niceperl** [comment]
I also got that error in my code (sqlite database).
The simplest code that fails:
`$dbix->table('test_table')->count`

Error: `DBD::SQLite::db prepare failed: near "FROM": syntax error at ...`

**Comment by alranel** [comment]
I just published DBIx::Lite 0.33 with a quick fix for this. It passes the failing test but I haven't tested it much on real applications. Let me know if it works for you
